src/openrct2-ui/windows/StaffList.cpp

```diff
@@ -176,9 +177,36 @@ rct_window* window_staff_list_open()
     window->max_height = MAX_WH;
     _quick_fire_mode = false;
 
+    WindowStaffListRefresh();
     return window;
 }
 
+void WindowStaffListRefresh()
```

Comment:
Wouldn't a static function on Window be better (`Window::RefreshStaffList()` or something)?

Reply:
What's our plan with Windows? Are we going to make each window a sub class of window?

Reply:
That would probably be the way to go. It would allow us to define invalidate/mouse-up/etc. functions without a lookup table, among other things.

Reply:
Yeah, I experimented with this a couple of years back:
https://github.com/IntelOrca/OpenRCT2/blob/refactor/window-3/src/openrct2-ui/windows/Cheats.cpp

Reply:
I'm going to leave this until the windows refactor then.